RCON Window Bugfixes

Bletch1971 opened the pull request:
1. Additional null checks added to prevent exceptions.

Bletch1971 committed:
RCON Window Bugfixes

1. Additional null checks added to prevent exceptions.

ChronosWS commented on ARK Server Manager/RCONWindow.xaml.cs:
```diff
@@ -506,7 +506,7 @@ public ICommand ChatPlayerCommand
                         CurrentInputWindowMode = InputWindowMode.ServerChatTo;
                         inputBox.Tag = player;
-                        inputTitle.Text = $"Send a chat message to : {player.SteamName}";
+                        inputTitle.Text = $"Send a chat message to : {player.SteamName ?? string.Empty}";
```
This is an odd case.  Can we put something like (unknown) in here instead of just a blank name if it is missing?

Also, localize

ChronosWS commented on ARK Server Manager/RCONWindow.xaml.cs:
```diff
@@ -528,13 +528,13 @@ public ICommand RenamePlayerCommand
                         CurrentInputWindowMode = InputWindowMode.RenamePlayer;
                         inputBox.Tag = player;
-                        inputTitle.Text = $"Rename player : from {player.ArkData.CharacterName}";
+                        inputTitle.Text = $"Rename player : from {player.ArkData.CharacterName ?? string.Empty}";
```
Localize and default (unknown or unnamed)

ChronosWS commented on ARK Server Manager/RCONWindow.xaml.cs:
```diff
@@ -550,7 +550,7 @@ public ICommand RenameTribeCommand
                         CurrentInputWindowMode = InputWindowMode.RenameTribe;
                         inputBox.Tag = player;
-                        inputTitle.Text = $"Rename tribe : from {player.TribeName}";
+                        inputTitle.Text = $"Rename tribe : from {player.TribeName ?? string.Empty}";
```
Localize, and add a better default (unknown or unnamed?)

Bletch1971 committed:
RCON Window Globalization

1. Added RCON strings into the language file.
2. Changed the RCON window to use the language file string.

Bletch1971 commented:
1. All RCON window strings have been converted to resource strings.
2. have changed blank string to unnamed.

Bletch1971 commented:
Cliff, any further changes?
